specification/resources/resource-manager/Microsoft.Authorization/stable/2020-05-01/examples/ManagementLocks_CreateOrUpdateAtResourceGroupLevel.json

```diff
@@ -0,0 +1,35 @@
+{
+  "parameters": {
+    "subscriptionId": "subscriptionId",
+    "resourceGroupName": "resourcegroupname",
+    "lockName": "testlock",
+    "api-version": "2016-09-01",
+    "parameters": {
+      "properties": {
+        "level": "ReadOnly"
+      }
+    }
+  },
+  "responses": {
+    "201": {
+      "body": {
+        "properties": {
+          "level": "ReadOnly"
+        },
+        "id": "/providers/Microsoft.Authorization/locks/testlock",
```

Comment:
ID should be based on the request scope (i.e. /subscriptions/subscriptionId/resourceGroups/resourcegroupname/providers/Microsoft.Authorization/locks/testlock). I see id is not correct in all examples. Better to fix them.